### DISCUSSION TITLE
How to Simulate Solid-Liquid Interaction Models ?

### DISCUSSION THREAD

**Discussion by buzhiqiuyuebai** [SeisSol/SeisSol#756] [Q&A] [answered]
I saw your example of tsunami simulation, [Verification of a 3D fully-coupled earthquake and tsunami model](https://github.com/SeisSol/Examples/tree/master/convergence_acoustic_elastic/convergence_ocean). But now I want to create a model similar to [WP2_LOH1](https://github.com/SeisSol/Examples/blob/master/WP2_LOH1), change the material of the upper layer to water(Physical Volume1), and the lower layer is rock(Physical Volume 2), I can in  the[ loh1.yaml file](https://github.com/SeisSol/Examples/blob/master/WP2_LOH1/loh1.yaml), make `mu=0, rho=1000,  lamda=2.25e9`, but as far as I know, solids and liquids follow different equations. How can I call different equations to calculate Volume1 and Volume2, Thanks in advance!
![image](https://user-images.githubusercontent.com/114474894/206686893-b18dec1e-e07a-4ae2-b99b-c592694d723b.png)

**Comment by krenzland** [comment, marked as answer]
Setting mu=0 is enough to use the acoustic wave equation.

**Reply by buzhiqiuyuebai** [reply]
Thank you for your kind help.
